app/src/pages/playground/PlaygroundOutput.tsx

```diff
@@ -187,6 +193,9 @@ function PlaygroundOutputText(props: PlaygroundInstanceProps) {
         providerKey: instance.model.provider,
         name: instance.model.modelName || "",
       },
+      invocationParameters: {
+        temperature: 0.1, // TODO: add invocation parameters
+      },
```

Comment:
should we start with 0.0?

Reply:
No temperature needs to be hard-coded in the frontend. I just have this here so I can test to make sure the invocation parameters are showing up in the instrumented LLM spans, but we will erase once we have some UI around it.